netsuite.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |gem|
   gem.version       = Netsuite::VERSION
 
   gem.add_dependency 'savon', '>= 2.3.0'
```

Comment:
We might want to pessimistically lock this. If `savon` releases a major version 3, we might not want to pick that up.